File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/SubscribeCommand.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class SubscribeCommand{

	@SerializedName("commandMetadata")
	private CommandMetadata commandMetadata;

	@SerializedName("subscribeEndpoint")
	private SubscribeEndpoint subscribeEndpoint;

	@SerializedName("clickTrackingParams")
	private String clickTrackingParams;

	public void setCommandMetadata(CommandMetadata commandMetadata){
		this.commandMetadata = commandMetadata;
	}

	public CommandMetadata getCommandMetadata(){
		return commandMetadata;
	}

	public void setSubscribeEndpoint(SubscribeEndpoint subscribeEndpoint){
		this.subscribeEndpoint = subscribeEndpoint;
	}

	public SubscribeEndpoint getSubscribeEndpoint(){
		return subscribeEndpoint;
	}

	public void setClickTrackingParams(String clickTrackingParams){
		this.clickTrackingParams = clickTrackingParams;
	}

	public String getClickTrackingParams(){
		return clickTrackingParams;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/SubscribeCommand.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import com.google.gson.annotations.SerializedName

class SubscribeCommand {
    @SerializedName("commandMetadata")
    var commandMetadata: CommandMetadata? = null

    @SerializedName("subscribeEndpoint")
    var subscribeEndpoint: SubscribeEndpoint? = null

    @SerializedName("clickTrackingParams")
    var clickTrackingParams: String? = null
    override fun toString(): String {
        return "SubscribeCommand{" +
                "commandMetadata = '" + commandMetadata + '\'' +
                ",subscribeEndpoint = '" + subscribeEndpoint + '\'' +
                ",clickTrackingParams = '" + clickTrackingParams + '\'' +
                "}"
    }
}